reference/library/src/morpheus-master/sass/oxford/_overrides.scss

```diff
@@ -78,6 +78,11 @@ $calendar-event-background-color: darken(#F8F8F8, 45%);
     }
 }
 
+img {
```

Comment:
@RebeccaOxfordUni Do you know where we should put stuff like this that isn't just for Oxford? We're not wanting to edit the raw bootstrap file ideally.